content/en/blog/2025/otel-js-sdk-2-0.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Announcing the OpenTelemetry JavaScript SDK 2.0
+linkTitle: OTel JS SDK 2.0
+date: 2025-03-21
+author: >
+  [Jamie Danielson](https://github.com/JamieDanielson) (Honeycomb)
+sig: OpenTelemetry JS
+cSpell:ignore: Danielson
+---
+
+Exciting news!
+[OpenTelemetry JavaScript](https://github.com/open-telemetry/opentelemetry-js)
+has released
+[SDK 2.0](https://github.com/open-telemetry/opentelemetry-js/releases)!
+
+There is a
+[migration guide](https://github.com/open-telemetry/opentelemetry-js/blob/main/doc/upgrade-to-2.x.md)
+in the repository detailing breaking changes and steps for users to take to
+upgrade to the latest versions.
```

Comment:
This paragraph seems redundant given the content below. If you intended this to be a TL;DR or call out, then wrap it in an alert like this:

```suggestion
{{% alert-md title="Migration guide" color="info" %}}

For a detailed description of breaking changes, see the **migration guide**
[Upgrade to OpenTelemetry JS SDK 2.x][migration guide].

[migration guide]:
  https://github.com/open-telemetry/opentelemetry-js/blob/main/doc/upgrade-to-2.x.md

{{% /alert-md %}}
```